dap/BESDapResponseBuilder.cc

```diff
@@ -1094,42 +1205,31 @@ void BESDapResponseBuilder::send_dap4_data_using_ce(ostream &out, DMR &dmr, bool
 
 void BESDapResponseBuilder::send_dap4_data(ostream &out, DMR &dmr, bool with_mime_headers)
 {
-    try {
```

Comment:
On January 4, 2016 at 5:52:20 PM, Patrick West (notifications@github.com) wrote:

In dap/BESDapResponseBuilder.cc:

> @@ -1094,42 +1205,31 @@ void BESDapResponseBuilder::send_dap4_data_using_ce(ostream &out, DMR &dmr, bool
>   
>  void BESDapResponseBuilder::send_dap4_data(ostream &out, DMR &dmr, bool with_mime_headers)
>  {
> -    try {
Is this part of the alarm branch? Or is it something else that probably should be in another branch?

Yes, it’s part of the branch. One request from NASA is that the timeout not include the time needed to transmit the result. so the alarm is turned on in BESInterface (and turned off there as well in the generic case) but optionally turned off in the dap module once it starts sending data. This is far from optimal since it introduces a complex kind of coupling between the operation of the framework and the handler, but I don’t see a way around that.


—
Reply to this email directly or view it on GitHub.


-- 
James Gallagher
jgallagher@opendap.org